contrib/depends/packages/freebsd_base.mk

```
@@ -1,6 +1,6 @@
 package=freebsd_base
 $(package)_version=11.3
-$(package)_download_path=https://download.freebsd.org/ftp/releases/amd64/$($(package)_version)-RELEASE/
+$(package)_download_path=http://ftp-archive.freebsd.org/mirror/FreeBSD-Archive/old-releases/amd64/$($(package)_version)-RELEASE/
```

```suggestion
$(package)_download_path=https://archive.freebsd.org/old-releases/amd64/$($(package)_version)-RELEASE/
```

`ftp-archive` redirected me to `archive` and 403'd

---

ftp-archive is 'not in the GeoDNS', for some reason "archive.freebsd" is not listed here https://docs.freebsd.org/en/books/handbook/mirrors/